app/scripts/framework/service/content-service.js

```diff
@@ -137,6 +137,8 @@ org.ekstep.services.contentService = new (org.ekstep.services.iService.extend({
 			instance.get(this.serviceURL() + this.getConfig('contentReadUrl', '/v3/read/') + contentId + metaDataFields, this.requestHeaders, function (err, res) {
 				/* istanbul ignore else */
 				if (res && res.data && res.data.responseCode === 'OK') {
+					// for telemetry 'ets' calculation - local time error correction
+					ecEditor.setContext('timeStamp', {serverTime: new Date(res.data.ts.slice(0, -1)), localTime: new Date()})
```

Comment:
1. This is not the right way to fetch the server time. Use the "date" response header to read server time.
2. Do not add this at the base method. It is called uneccessarily for every api call
3. Where is the corresponding change in the player/renderer

@loganathan1989 @pallakartheekreddy